testdata/react/src/App.js

```diff
@@ -0,0 +1,15 @@
+import React, { Component } from 'react';
```

Comment:
Please include steps to regenerate this project e.g. when we want to use a different version of webpack or react, or make any changes. e.g. other test projects have a tsconfig.json included and I can just run `tsc` to rebuild them.

Reply:
Will do. I'm planning on creating a separate repo to house the test project code so as to not weigh down this one with a bunch of react/angular/etc. dependencies

Reply:
Honestly I think it would be fine to just check in the package.json but not the node_modules. Then we can build/update it in place in case we want to change anything. Seems the simpler than having to find another repo and copy files around?

Reply:
True. I was just worried about adding a lot of bloat to the repo, but if it works for you I'll just add the whole thing!

Reply:
Added the full project under testData/react